packages/eslint-config-sevenval/package.json

```diff
@@ -23,6 +23,6 @@
     "eslint-plugin-flowtype": "^2.0.0",
     "eslint-plugin-import": "^2.13.0",
     "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-react": "^7.9.1"
+    "eslint-plugin-react": "^7.12.4"
```

Comment:
`yarn.lock` should also be updated. This can be done via a simple `yarn` command from the repo's root folder.

Reply:
done